src/helper.js

```diff
@@ -323,16 +319,16 @@ export const getRootGroupByModel = (model) => {
 };
 
 export const getRootDataModel = (model) => {
-    if (model._parent) {
-        return getRootDataModel(model._parent);
```

Comment:
Why change the method?

Reply:
Just a small code refactoring as a bonus, here looping is better than recursive function